mindsworth/My-Diary-Frontend
# #162363108 Setup react router

#### What does this PR do?
Setup routing for the app with react-router version 4

#### Description of Task to be completed?
- installed react-router-dom
- installed history for browser histories

#### How should this be manually tested?
1. After cloning or pulling the branch to your local
2. Run yarn install to install all necessary packages and run yarn start:dev start your app on port 8080
3. visit the right URL e.g http://localhost:8080 to see the routing links at the top.

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
-[ #162363108](https://www.pivotaltracker.com/story/show/162363108)

#### Screenshots (if appropriate)
<img width="1680" alt="screenshot 2018-12-01 at 3 48 15 pm" src="https://user-images.githubusercontent.com/14542449/49329524-94733f80-f580-11e8-9981-80e2defa42ec.png">

#### Questions:
N/A
